src/exberry_int/integration_exberry.py

```diff
@@ -191,27 +199,37 @@ async def handle_create_instrument_request(event):
 
     @events.queue.message()
     async def process_inbound_messages(msg):
-
-        if msg['q'] == EXBERRY_ORDERBOOK_DEPTH and msg['d']['messageType'] == 'Executed':
+        if 'errorType' in msg:
+            error_type = msg['errorType']
+            error_code = msg['d']['errorCode']
+            error_message = msg['d']['errorMessage']
+            LOG.error(f'Encountered erorr - Type: {error_type}, Code: {error_code}, Message: {error_message}')
+            if error_code == '1200':
```

Comment:
What is the significance of 1200 and why is this number represented as a string?

Reply:
1200 is the "General Error" code that is killing the orderbook subscription. The number should be a number, I was confusing it with `errorType` which comes in as a number as a string. Will fix, thank you.